core/server/src/proto/journal/file.proto

```diff
@@ -67,7 +67,12 @@ message InodeDirectoryIdGeneratorEntry {
   optional int64 sequence_number = 2;
 }
 
-// next available id: 17
+enum PTtlAction {
```

Comment:
I think enum makes sense for configurations like Ttl actions, where the permitted values are limited (0/1). Even enums are encoded as i32.

**Note:** I have still not changed the thrift/proto definitions, as part of this commit. We can discuss, if we agree on non-enum, then I'll change